features/index_page.feature

```diff
@@ -0,0 +1,12 @@
+Feature: index page with a list restaurants
+  As a visitor
+  As soon as I arrive at the index page
```

Comment:
Yes, so you could write something like:
```
As a visitor
in order to access the app
I need to see an index page
```
or `in order to view some food to buy` or some other thing. `As soon as I arrive at the index page` is not a business objective.